src/styles/blocks/main-page.scss

```diff
@@ -0,0 +1,8 @@
+.main-page {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  &__content {
```

Comment:
will be better to reset margins for body in reset.scss with all other elements

Reply:
agree